docs/build/linux-ubuntu.md

```diff
@@ -64,3 +64,23 @@ All dependencies are already installed if you followed the instructions under [P
 Building:
 
 * [Building Keyman Core](../../core/doc/BUILDING.md)
+
+## Docker Builder
+
+The Docker builder allows you to perform a linux build from anywhere Docker is supported.
+To build the docker image:
+
+```shell
+cd ../../linux
+docker pull ubuntu:latest
+docker build . -t keymanapp/keyman-linux-builder:latest
+```
+
+Once the image is built, it may be used to build parts of Keyman.
+
+```shell
+cd ../core
+# keep linux build artifacts separate
+mkdir -p build/linux
+docker run -it --rm -v $(pwd)/..:/home/build -v $(pwd)/build/linux:/home/build/core/build keyman-linux-builder:latest bash -c 'cd core; bash build.sh -d'
```

Comment:
Do we need both `-v`s? I'd think the latter is included in the former.

Reply:
no, by default it builds into `build/arch` which would stomp on your local build.   This creates a `build/linux` which is its own `build` subdirectory.